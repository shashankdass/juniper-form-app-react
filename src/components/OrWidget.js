import React, {Component} from 'react';
import AndWidget from './AndWidget'
class OrWidget extends Component {
	render() {
		const headingStyle = {
			marginLeft: "25%"
		};
		return (
			<div>
        <h3 style={headingStyle}> Or </h3>
				<AndWidget key={this.props.key} uniKey={this.props.uniKey} />
			</div>
		)
	}
}

export default OrWidget;
