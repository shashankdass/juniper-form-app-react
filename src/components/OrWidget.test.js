import React from 'react';
import OrWidget from './OrWidget';
import renderer from 'react-test-renderer';

test('it renders OrWidget Component', () => {
  const component = renderer.create(
    <OrWidget></OrWidget>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
