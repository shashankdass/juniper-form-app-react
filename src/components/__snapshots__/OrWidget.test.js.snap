exports[`test it renders OrWidget Component 1`] = `
<div>
  <h3
    style={
      Object {
        "marginLeft": "25%",
      }
    }>
     Or 
  </h3>
  <div
    style={
      Object {
        "padding": 10,
      }
    }
    uniKey={undefined}>
    <div
      className="col">
      <label
        className=" form-control-label"
        htmlFor={undefined}
        style={
          Object {
            "padding": 10,
          }
        }>
        Name:
         
        <input
          name="whereName_undefined"
          type="text" />
      </label>
          
      <label
        className=" form-control-label"
        htmlFor={undefined}
        style={
          Object {
            "padding": 10,
          }
        }>
        Value:
         
        <input
          name="whereValue_undefined"
          type="text" />
      </label>
          
      <label
        className=" form-control-label"
        htmlFor={undefined}
        style={
          Object {
            "padding": 10,
          }
        }>
        Operator:
         
        <select
          name="whereOperator_undefined">
          <option
            value="empty" />
          <option
            value="=">
            =
          </option>
          <option
            value="!=">
            !=
          </option>
        </select>
      </label>
    </div>
  </div>
</div>
`;
